x11-packages/far2l/build.sh

```diff
@@ -0,0 +1,19 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/elfmz/far2l
+TERMUX_PKG_DESCRIPTION="FAR Manager v2 - Linux port"
```

Comment:
```suggestion
TERMUX_PKG_DESCRIPTION="FAR Manager v2"
```
I think a general description is enough